guides/_subsections/automated-analysis.md

```diff
@@ -54,25 +50,39 @@ for offline use.
     %}
   </li>
   <li>
+    {% capture download-report-text %}
+    <p>
+      <figure>
+        <a href="{{ site.url }}/images/2.2.0/automated-analysis-4.png" target="_blank">
+          <img src="{{ site.url }}/images/2.2.0/automated-analysis-4.png">
+        </a>
+        <figcaption>
+          View the report on its own without connecting to Cryostat.
+          Check <i>Show OK Results</i> to include results where the rules
+          engine found no issues in the recording.
+        </figcaption>
+      </figure>
+      <figure>
+        <a href="{{ site.url }}/images/2.2.0/automated-analysis-5.png" target="_blank">
+          <img src="{{ site.url }}/images/2.2.0/automated-analysis-5.png">
+        </a>
+        <figcaption>
+          To download the HTML <i>Automated Analysis</i> report to local disk, right click 
+          the page and select <i>Save Page As...</i>. Alternatively, press <kbd>Ctrl</kbd>
+          +<kbd>S</kbd> on Windows/Linux, or <kbd>⌘</kbd>+<kbd>S</kbd> on macOS.
```

Comment:
The css works nice :D Probably remove the extra space after `Ctrl`? Or add one between all?

![Screenshot from 2022-11-02 14-35-09](https://user-images.githubusercontent.com/68053619/199573430-ea906706-04e3-4933-a209-12285701e454.png)